### PR TITLE
Add information about fontSize option

### PR DESCRIPTION
**Pull request by i2amsam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ When using `frameit` without titles on top, the screenshots will have the full r
 
 # Titles and Background (optional)
 
-With `frameit` 2.0 you are now able to add a custom background, title and text colors to your screenshots.
+With `frameit` 2.0 you are now able to add a custom background, title and text colors to your screenshots.  You can also optionally set the fontSize of text, or leave it out and let `frameit` resize the text automatically.
```
Comment (KrauseFx) [inline README.md]: 2 spaces after `.` :cry:

Comment (KrauseFx) [inline README.md]: How about just saying

> If you want, you can also set a custom font size

Comment (i2amsam) [inline README.md]: Sounds good, updated!

**Comment by KrauseFx** [comment]
I'd prefer an easier and shorter sentence, but this works for me

**Pull request merged**